### PR TITLE
docs(i18n): add Polish localization scout

### PR DESCRIPTION
**Pull request by Dante-dan**
## What Changed

- Added a bounded Polish scout under `docs/pl/`: a Polish getting-started and core-concepts guide plus a terminology glossary.
- Registered `docs-pl` and `locale:pl`, with both `pl` and `pl-PL` installer aliases, and included the locale in the npm publish surface.
- Added Polish to the language navigation in the root and existing localized READMEs.
- Extended the existing locale-install contract checks for Polish module resolution and the regional alias.

The Polish guide records English source commit `8321021c` and its exact translated scope. It links the complete English README rather than presenting this first increment as a full translation.

## Why This Change

This implements the scout proposed by @majevva in #3096 and follows their correction to use the short `pl` directory code. It also follows the owner guidance and merged German precedent in #1980/#2029: keep new locale docs under `docs/<locale>/`, establish terminology, make the locale discoverable, and register installer support in the same bounded PR.

The initial guide deliberately focuses on setup, the major ECC surfaces, safety, and the translation workflow. Further translation can continue incrementally by domain without creating another large overlapping localization PR.

Refs #3096

## Testing Done

- [ ] Manual testing completed
- [ ] Automated tests pass locally (`node tests/run-all.js`)
- [x] Edge cases considered and tested

Focused repository checks completed:

- `node tests/lib/locale-install.test.js` — 12 passed, 0 failed
- `node tests/lib/install-manifests.test.js` — 41 passed, 0 failed
- `node scripts/ci/validate-install-manifests.js` — validated 38 modules, 85 components, and 7 profiles
- `node tests/scripts/npm-publish-surface.test.js` — 2 passed, 0 failed
- `markdownlint 'docs/pl/**/*.md'` — clean
- `git diff --check` — clean

The aggregate `npm test` was not run because this documentation/manifest slice was covered by the focused checks above and the repository aggregate currently includes local UI/E2E paths outside this run's allowed validation scope.

## Type of Change

- [ ] `fix:` Bug fix
- [x] `feat:` New feature
- [ ] `refactor:` Code refactoring
- [x] `docs:` Documentation
- [x] `test:` Tests
- [ ] `chore:` Maintenance/tooling
- [ ] `ci:` CI/CD changes

## Security & Quality Checklist

- [x] No secrets or API keys committed (ghp_, sk-, AKIA, xoxb, xoxp patterns checked)
- [x] JSON files validate cleanly
- [ ] Shell scripts pass shellcheck (if applicable) — no shell changes
- [ ] Pre-commit hooks pass locally (if configured) — not run
- [x] No sensitive data exposed in logs or output
- [x] Follows conventional commits format

## If you changed dependencies or `package.json` (`bin` / `files` / deps)

- [x] Ran `yarn install --mode=update-lockfile`; `yarn.lock` remained unchanged.

## If you added a skill, command, agent, hook, or CLI tool

Not applicable; this adds a documentation locale and installer manifest entries.

## Documentation

- [x] Updated relevant documentation
- [ ] Added comments for complex logic — no complex logic added
- [x] README updated (if needed)

### CODE REVIEW

File: manifests/install-modules.json [hunk 1]
```diff
@@ -1177,6 +1177,22 @@
       "defaultInstall": false,
       "cost": "heavy",
       "stability": "stable"
+    },
+    {
+      "id": "docs-pl",
+      "kind": "docs",
+      "description": "Polish (pl) translated reference docs for agents, commands, skills, and rules.",
```
[the remark this answers is not in the document]
Comment (Dante-dan) [reply manifests/install-modules.json]: Good catch—the module description overstated the translated surface. I've narrowed it to the getting-started/core-concepts guide and terminology glossary that `docs/pl` actually ships. The install manifest validator, install-manifests tests (41/41), and locale-install tests (12/12) pass.

A broader local repository run also reached three failures outside this metadata path: the shared TypeScript build in `opencode-legacy-migration`, a sandbox-denied write to `~/.claude/package-manager.json`, and a PowerShell timing case. The targeted manifest/locale checks passed; the broader run was stopped after those failures.

**Comment by Dante-dan** [comment]
@affaan-m Two current-head CI failures match the default-branch baseline and are outside this PR's net diff.

### Lint

- PR head `7371f035`: [run 35687053056](https://github.com/affaan-m/ECC/actions/runs/35687053056), [Lint job/log](https://github.com/affaan-m/ECC/actions/runs/35687053056/job/106616018866)
- `main` at `bf70150e`: [run 35639889305](https://github.com/affaan-m/ECC/actions/runs/35639889305), [Lint job/log](https://github.com/affaan-m/ECC/actions/runs/35639889305/job/106466570287)

Both Lint jobs report the same two ESLint errors: unused `spawn` in `scripts/control-pane.js:4` and undefined `spawn` in `scripts/plan-canvas.js:182`.

### Ubuntu Node 20 / Bun test suite

- PR head `7371f035`: [run 35687053056](https://github.com/affaan-m/ECC/actions/runs/35687053056), [test job/log](https://github.com/affaan-m/ECC/actions/runs/35687053056/job/106616019397) — 4,886 passed, 25 failed
- `main` at `bf70150e`: [run 35639889305](https://github.com/affaan-m/ECC/actions/runs/35639889305), [test job/log](https://github.com/affaan-m/ECC/actions/runs/35639889305/job/106466570574) — 4,882 passed, 25 failed

The two test jobs have the same 25 named failures and matching error signatures: nine `plan-canvas-e2e` failures after the detached server fails to start (including `ECONNREFUSED`), three `claude-scope-migration` assertions, ten pre-push/pytest assertions, and one failure each in `control-pane`, `install-apply`, and `setup`.

The failing source and test paths are identical to `main` in this PR. I left these unrelated baseline files untouched; the Polish locale's focused manifest, install, publish-surface, and Markdown checks pass.

**Comment by chenhz01** [comment]
The scout follows the #1980/#2029 precedent cleanly, and the CI-baseline note in the thread covers the pre-existing failures. One structural gap: the installer alias and contract checks for Polish were added by hand, per-locale. Nothing generic enforces that every `docs/<locale>/` directory has a matching `locale:<code>` registration and alias pair — the next locale scout can ship docs without installer support and CI stays green, which is exactly the drift the German precedent was meant to prevent. Suggest a contract test that derives the expectation instead of asserting it:
```js
for (const dir of fs.readdirSync('docs').filter(isLocaleDir))
  assert(aliasExists(dir) && manifestRegisters(dir));
```
That turns the per-locale checks extended for `pl` into a loop covering `ur`-style short codes and `de-DE`-style regional ones uniformly. Happy to open a PR with working code if useful.
